src/main/java/com/coveo/saml/SamlClient.java

```diff
@@ -95,14 +96,16 @@ public void setDateTimeNow(DateTime now) {
    * @param responseIssuer              the expected issuer ID for SAML responses.
    * @param certificates                the list of base-64 encoded certificates to use to validate
    *                                    responses.
+   * @param useRedirect                 if client should use HTTP-Redirect instead of HTTP-POST
    * @throws SamlException thrown if any error occur while loading the provider information.
    */
   public SamlClient(
       String relyingPartyIdentifier,
       String assertionConsumerServiceUrl,
       String identityProviderUrl,
       String responseIssuer,
-      List<X509Certificate> certificates)
+      List<X509Certificate> certificates,
+      Boolean useRedirect)
```

Comment:
Maybe an enum would have been better here?

Reply:
Yeah, probably so.  I'll take a stab at that.

Reply:
Okay, reworked this as an enum.